src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -116,6 +116,14 @@ public function getRouteCollectionForRequest(Request $request)
                         )
                     );
                 } elseif (
+                    $content->getNodeType() === Structure::NODE_TYPE_EXTERNAL_LINK &&
+                    $content->getNodeState() === StructureInterface::STATE_PUBLISHED
+                ) {
+                    $collection->add(
+                        $content->getKey() . '_' . uniqid(),
+                        $this->getRedirectRoute($request, 'http://' . $content->getPropertyValue('external'))
```

Comment:
How do you know the property name is always going to be `external`? it depends on the form does it not?